### PR TITLE
Add 'You are here' marker

### PR DESCRIPTION
**Pull request by DanielHeath**

### CODE REVIEW

**Comment by danni** [comment]
Test?

**Comment by DanielHeath** [comment]
Done. ++?

File: test/features/10-functional/30-maps/10-results-map.feature [hunk 1]
```diff
@@ -33,6 +33,7 @@ Feature: See results on a map
         ------------------------------------------
         Title              | Latitude | Longitude
         ==========================================
+        You are here       | -37.8220  | 144.990
```
Comment (danni) [inline test/features/10-functional/30-maps/10-results-map.feature]: Random extra space? :-P

**Comment by danni** [comment]
++ but fix space

**Pull request merged**